emails/tracker_lists/level-two-tracker.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+["1-2-1marketing.com", "accelo.com", "mkt10008.com", "mkt10039.com", "mkt10049.com", "mkt10067.com", "mkt10114.com", "mkt10153.com", "mkt10663.com", "mkt10781.com", "mkt1248.com", "mkt1365.com", "mkt1937.com", "mkt1946.com", "mkt2178.com", "mkt2478.com", "mkt2685.com", "mkt2724.com", "mkt32.net", "mkt3469.com", "mkt3536.com", "mkt3797.com", "mkt3798.com", "mkt3838.com", "mkt4091.com", "mkt41.net", "mkt4158.com", "mkt4261.com", "mkt4424.com", "mkt4463.com", "mkt4477.com", "mkt4644.com", "mkt5089.com", "mkt51.net", "mkt5216.com", "mkt5224.com", "mkt5297.com", "mkt5379.com", "mkt5419.com", "mkt5514.com", "mkt5566.com", "mkt5654.com", "mkt5657.com", "mkt5906.com", "mkt6031.com", "mkt61.net", "mkt6260.com", "mkt6264.com", "mkt6288.com", "mkt6316.com", "mkt6478.com", "mkt6688.com", "mkt6735.com", "mkt6793.com", "mkt685.com", "mkt6882.com", "mkt6903.com", "mkt6917.com", "mkt6967.com", "mkt71.net", "mkt7234.com", "mkt7580.com", "mkt7596.com", "mkt7752.com", "mkt7783.com", "mkt7832.com", "mkt7842.com", "mkt7883.com", "mkt7946.com", "mkt7971.com", "mkt7972.com", "mkt7974.com", "mkt8007.com", "mkt8008.com", "mkt8043.com", "mkt8062.com", "mkt8063.com", "mkt8064.com", "mkt8096.com", "mkt81.net", "mkt8133.com", "mkt8163.com", "mkt8267.com", "mkt829.com", "mkt8345.com", "mkt8369.com", "mkt8586.com", "mkt8628.com", "mkt8756.com", "mkt8763.com", "mkt8988.com", "mkt9026.com", "mkt9054.com", "mkt912.com", "mkt9203.com", "mkt922.com", "mkt941.com", "mkt9430.com", "mkt9775.com", "mkt9862.com", "mkt9923.com", "mkt9942.com", "acemlna.com", "acemlnb.com", "acemlnc.com", "acemlnd.com", "activehosted.com", "emlnk1.com", "acsmedia.us", "activedemand.com", "adobe.com", "agilecrm.com", "aisleahead.com", "atompark.com", "autoklose.com", "bitrix24.com", "exponea.com", "getblueshift.com", "boomeranggmail.com", "bounceexchange.com", "campaignmonitor.com", "carrotquest.io", "cirrusinsight.com", "connectif.cloud", "rs6.net", "customeriomail.com", "didtheyreadit.com", "buttondown.email", "emailtracker.website", "engagebay.com", "ezymarketer.net", "emailinc.net", "followup.cc", "getnotify.com", "gliq.com", "gmelius.com", "hunter.io", "lahar.com.br", "lassocrm.com", "leadboxer.com", "bra2hmail.com", "mailcamp.nl", "list-manage.com", "tinyletter.com", "mailerjet.com", "mailerlite.com", "mailspice.com", "mailtag.io", "mailtrack.io", "mixmax.com", "moengage.com", "outreach.io", "persistiq.com", "polymail.io", "propelleremail.co.uk", "responder.co.il", "send24.pl", "reply.io", "retailrocket.net", "revenuegrid.com", "smartcloudconnect.io", "rightinbox.com", "sailthru.com", "exacttarget.com", "pardot.com", "saleshandy.com", "salesloft.com", "streak.com", "substack.com", "tradiewebguys.com.au", "sendgrid.net", "user.com", "visme.co", "vocus.io", "woowup.com", "wordfly.com", "yesware.com", "yotpo.com", "zendable.com", "maillist-manage.com", "maillist-manage.com.au", "maillist-manage.eu", "maillist-manage.in"]
```

Comment:
Can you run this file through a JSON pretty-printer like `jq`?

Reply:
[cbdab0c](https://github.com/mozilla/fx-private-relay/pull/2268/commits/cbdab0c7c897cf9f2b12e70f8d680e2beffb3c25)